data/scripts/sol-oic-gen.py

```diff
@@ -480,7 +480,7 @@ def object_inform_flow_fn_common_c(state_struct_name, name, props, client):
                 'field_name': field_name
             }
 
-        send_flow_pkts.append('''%(flow_send_fn)s(resource->node, SOL_FLOW_NODE_TYPE_%(STRUCT_NAME)s__OUT__OUT_%(FIELD_NAME)s, %(val)s);''' % {
+        send_flow_pkts.append('''%(flow_send_fn)s(resource->node, SOL_FLOW_NODE_TYPE_%(STRUCT_NAME)s__OUT__%(FIELD_NAME)s, %(val)s);''' % {
```

Comment:
I still see ports with double IN/OUT out there -- we gotta fix these leftovers as well.